include/aspect/utilities.h

```diff
@@ -1217,6 +1217,60 @@ namespace aspect
       });
       return sorted_vec;
     }
+
+    /**
+     * Contains utility functions related to tensors.
+     */
+    namespace Tensors
+    {
+
+      /**
+       * Rotate a 3D 4th order tensor with an other 3D 2th order tensor
+       */
+      SymmetricTensor<4,3>
+      rotate_4th_order_tensor(const SymmetricTensor<4,3> &input_tensor, const Tensor<2,3> &rotation_tensor);
+
+
+      /**
+       * Rotate a 6x6 voigt matrix with an other 3D 4th
+       */
+      SymmetricTensor<2,6>
+      rotate_6x6_matrix(const SymmetricTensor<2,6> &input_tensor, const Tensor<2,3> &rotation_tensor);
+
+      /**
+       * Transform a 4th order tensor into a 6x6 matrix
```

Comment:
Period at the end of the sentence for all of the doc strings of all of these functions.